src/app/templates/app/base.html

```diff
@@ -181,7 +181,7 @@
     </div><!--/.nav-collapse -->
   </nav>
 
-    <div class="container">
+    <div class="container body1">
```

Comment:
I see this in base.html
```
<div class="container">

      <div class="starter-template">
        {% block body1 %}{% endblock %}
      </div>

    </div><!-- /.container -->
		{% block body2 %}{% endblock %}
```

submit_new_license is made with body2. Can you try moving it to body1 coz body1 will always be present and take up some space. For me deleting the whole container element on this page gave me least space. If we use `body1` block itself for this page then it would be similar to other pages.

Reply:
Hi @rtgdk , I tried putting the page itself in `block1` but the problem is that the block one is specifically only being used for the modal, because of which it doesn't take up the full space and instead the page looks like
![image](https://github.com/spdx/spdx-online-tools/assets/65075935/34e30b84-8885-4f3b-b12e-e1007b28b769)
So in the current code I have made use of `block2` let me know if anything in that needs to be changed